wikimedia/wikipedia-ios
# Watchlist – [3] Update filter view when new language is added

**Phabricator:** https://phabricator.wikimedia.org/T356794 (partially)

### Notes

This is part of a series of PRs that adds the functionality to add new languages from the Watchlist Filter view. This PR replaces the previously presented language view controller, allows a user to select a new preferred language to add to the app/Watchlist, and reloads the Filter view's Wikipedias section.

### Test Steps

1. Open Watchlist
2. Open the Filter View
3. Tap the Add Language accessory row
4. Select a new language
5. Confirm that language is presented in the Wikipedia sections view
6. Confirm dismissing the filter view shows the articles in that new language you're watching (since my watchlist items are primarily English, I watched an article in a different language, removed that language from my preferred languages, tested adding that language via this Filter view to confirm those articles displayed in the main watchlist view)
